packages/components/tooltip/src/tooltip.styles.js

```diff
@@ -2,20 +2,6 @@ import styled from '@emotion/styled';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 import Constraints from '@commercetools-uikit/constraints';
 
-const getOffsetMargin = ({ placement }) => {
```

Comment:
This is no longer needed because the margin offset is now set directly through the proper popper `offset` modifier.